README.md

```diff
@@ -222,7 +222,7 @@ We are looking for maintainers to add more parsers and to write query files for
 - [x] [func](https://github.com/amaanq/tree-sitter-func) (maintained by @amaanq)
 - [x] [fusion](https://gitlab.com/jirgn/tree-sitter-fusion.git) (maintained by @jirgn)
 - [x] [Godot (gdscript)](https://github.com/PrestonKnopp/tree-sitter-gdscript) (maintained by @PrestonKnopp)
-- [x] [gitconfig](https://github.com/the-mikedavis/tree-sitter-git-config) (maintained by @amaanq)
+- [x] [git_config](https://github.com/the-mikedavis/tree-sitter-git-config) (maintained by @amaanq)
```

Comment:
The readme list is auto-generated. You must also change parsers.lua. Parsers can specify and alternative name in the readme

Reply:
> The readme list is auto-generated. You must also change parsers.lua. Parsers can specify and alternative name in the readme

Okay, I have updated parsers.lua to reflect the readme.